ibtk/include/ibtk/private/StreamableManager-inl.h

```diff
@@ -111,7 +111,7 @@ StreamableManager::unpackStream(SAMRAI::tbox::AbstractStream& stream,
     {
         data_items[k] = unpackStream(stream, offset);
     }
-    std::vector<SAMRAI::tbox::Pointer<Streamable> >(data_items).swap(data_items); // trim-to-fit
+    data_items.shrink_to_fit(); // trim-to-fit
```

Comment:
Probably can eliminate this comment now that the function name says what it does.